### PR TITLE
Cleanup Cosmos Go Deps

### PR DESCRIPTION
**Pull request by JonathanLorimer**

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -119,8 +119,8 @@
           gaia6 = mkApp { name = "gaia"; drv = packages.gaia6_0_4; exePath = "/bin/gaiad"; };
           gaia6_0_2 = mkApp { name = "gaia"; drv = packages.gaia6_0_2; exePath = "/bin/gaiad"; };
           gaia6_0_3 = mkApp { name = "gaia"; drv = packages.gaia6_0_3; exePath = "/bin/gaiad"; };
```
Comment (adizere) [inline flake.nix:121]: Note that Gaia v4 is deprecated because SDK v0.42 reached end of life.

Comment (JonathanLorimer) [reply flake.nix:118]: Oh, good to know. Removed.

File: resources.nix [hunk 2]
```diff
@@ -15,72 +15,76 @@ let
     inherit pkgs inputs;
   };
 
+  ibc-packages = import ./resources/ibc-go {
+    inherit pkgs inputs;
+  };
+
   # Cosmos packages
-  packages = rec {
+  packages = with inputs; rec {
```
Comment (soareschen) [inline resources.nix:23]: I consider the `with` statement as being an [antipattern](https://nix.dev/anti-patterns/language#with-attrset-expression) that should be avoided. It makes the source code difficult to parse to find out which variable is coming from where.

Comment (JonathanLorimer) [reply resources.nix:23]: I think its fine when its localized and the context is kindof implicit. But I am happy to revert

Comment (soareschen) [reply resources.nix]: Would be good to revert. The scope here is pretty large, and reader have to look quite far ahead to find out the with statement.

**Review by soareschen — APPROVED** [review]

**Pull request merged**